### PR TITLE
Feature/with blog

### PR DESCRIPTION
**Pull request by margaritis**
Add authors on config
Add new inlcudes ( post, disqus, social)
Add new layouts ( page, post, blog index)
Bring analytics tracker on config
Split index to components
Minor other changes
New blog post -> All about DEVit

### CODE REVIEW

File: app/css/devit-custom.css [hunk 1]
```diff
@@ -165,3 +165,24 @@ p.cfp-moto {
...
+.post h1{
+  font-size: 6rem;
+}
+
+.share-page {
+    text-align: center;
+    padding: 8px 15px;
+    border-radius: 5px;
+
+    a {
```
Comment (kbariotis) [inline app/css/devit-custom.css]: This is a CSS file. Not SASS. :P

File: app/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title : Blog
+header : DEVit Conference Blog
+---
+{% assign authors = site.authors %}
+
+
+<div class="posts">
```
Comment (kbariotis) [inline app/blog/index.html]: We should a date here.

File: app/_includes/post.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<section id="post">
+
+  <article class="post row">
+
+    <div class="page-header">
+      <h1>{{ page.title }}</h1>
```
Comment (kbariotis) [inline app/_includes/post.html]: Make this the same style like the tiles in the blog index.

File: app/blog/index.html (hunk 2 above)
Comment (margaritis) [inline app/blog/index.html]: What do you mean? Probably add a date on each post after the name? Right?

File: app/_includes/post.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<section id="post">
+
+  <article class="post row">
+
+    <div class="page-header">
```
Comment (kbariotis) [inline app/_includes/post.html]: I think we should keep the header here. Just for the aesthetics. I know it's usable but without the header is even worse.

File: app/blog/index.html (hunk 2 above)
Comment (kbariotis) [inline app/blog/index.html]: Yes sorry, add the date when the post written.

Comment (margaritis) [inline app/blog/index.html]: :+1:

File: app/css/devit-custom.css (hunk 1 above)
Comment (margaritis) [inline app/css/devit-custom.css]: I will add the CSS thingy.

**Comment by margaritis** [comment]
I think it is good to go.

**Comment by ntemposd** [comment]
WoW! this was too fast :smile:

**Pull request merged**